crates/sui-framework/packages/deepbook/sources/custodian.move

```diff
@@ -37,6 +37,9 @@ module deepbook::custodian {
         custodian: &Custodian<Asset>,
         user: ID
     ): (u64, u64) {
+        if (!table::contains(&custodian.account_balances, user)) {
```

Comment:
Same here--please separate into a separate PR with the test.

Reply:
Good suggestion, I will do that!